core/src/saros/activities/TextSelectionActivity.java

```diff
@@ -46,15 +46,29 @@
   @XStreamAsAttribute
   private final int endInLineOffset;
 
+  @XStreamAlias("bs")
+  private final boolean isBackwardsSelection;
+
+  /**
+   * Calls {@link #TextSelectionActivity(User, TextSelection, boolean, IFile)} with <code>
+   * isBackwardsSelection=false</code>.
+   */
+  public TextSelectionActivity(User source, TextSelection selection, IFile file) {
+    this(source, selection, false, file);
+  }
+
   /**
    * Instantiates a new text selection activity.
    *
    * @param source the user that created the selection
    * @param selection the text selection
    * @param file the file the text was selected in
+   * @param isBackwardsSelection whether the selection is a backwards selection
    * @throws IllegalArgumentException if the source or file is <code>null</code>
    */
-  public TextSelectionActivity(User source, TextSelection selection, IFile file) {
+  public TextSelectionActivity(
```

Comment:
The boolean `isBackwardsSelection` seems to be a property of TextSelection. What do you think about moving it into TextSelection (with default value `false`)?

Reply:
I though about adding it there but decided against it as `TextSelection` is used a lot besides the text selection activity logic. But I can change it.

Reply:
Added to `TextSelection`.